_std/defines/item.dm

```diff
@@ -44,6 +44,8 @@
 #define TGUI_INTERACTIVE		 (1<<19)
 /// Has a click delay for attack_self()
 #define ATTACK_SELF_DELAY		 (1<<20)
+///can be put in the glove slot
+#define ONGLOVE (1<<21)
```

Comment:
don't know that this really need to be a bit flag. Why not just make the bracelets a subtype of glove with `hide_prints = 0`

Reply:
hmm, ill try it

Reply:
done!